src/services/userService.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const { User } = require('../models/User.model');
+
+const userService = {
+  async createUser({ name }) {
+    const newUser = await User.create({ name });
+
+    return newUser;
+  },
+
+  async getAllUsers(filters = {}) {
+    const where = {};
+
+    if (filters.id) {
+      where.id = filters.id;
+    }
+
+    if (filters.name) {
+      where.name = filters.name;
+    }
+
+    const users = await User.findAll({ where });
```

Comment:
Seens simply replacing `filters = {}` with `where = {}` and removing the _ifs_ would get the same result